Docker/Dockerfile

```
@@ -4,8 +4,8 @@
 FROM oraclelinux:9
 
 LABEL author="Geoscience Australia" \
-description="ISCE3/RTC Pipeline Version 0.1" \
-version="0.1"
+description="ISCE3/RTC Pipeline Version 0.2.1_beta" \
```

```suggestion
description="ISCE3/RTC Pipeline Version 0.2.2_beta" \
```

Should the version numbers in the description and line below match?